admin/src/js/services/select2-search.js

```diff
@@ -114,26 +127,72 @@ angular.module('inboxServices').factory('Select2Search',
           });
       };
 
-      const resolveInitialValue = function(selectEl, initialValue) {
+      const addValue = (val) => {
+        if (!selectEl.children('option[value="' + val + '"]').length) {
+          selectEl.append($('<option value="' + val + '"/>'));
+        }
+      };
+
+      const getDocs = function (ids) {
+        const docsFetched = 10;
```

Comment:
It's unclear why we are limiting to 10 documents, when we can clearly have a longer selection.